backend/workflow_manager/workflow/workflow_helper.py

```diff
@@ -144,30 +144,32 @@ def process_input_files(
                     single_step=single_step,
                     file_hash=file_hash,
                 )
-                if is_executed:
-                    processed_files += 1
                 if error:
-                    error_raised += 1
+                    failed_files += 1
+                else:
+                    successful_files += 1
             except StopExecution as exception:
                 execution_service.update_execution(
                     ExecutionStatus.STOPPED, error=str(exception)
                 )
                 break
             except Exception as error:
```

Comment:
@chandrasekharan-zipstack Can we change error as e here? It is confusing because same variable name is declared above